config/config.go

```diff
@@ -162,7 +163,15 @@ func (cfg *Config) dsn() string {
 		sslMode = "verify-full"
 	}
 
-	return fmt.Sprintf("postgres://%s@%s:%d/%s?sslmode=%s", url.UserPassword(cfg.db.User, cfg.db.Pass), cfg.db.Host, cfg.db.Port, cfg.db.Schema, sslMode)
+	return fmt.Sprintf(
+        "%s://%s@%s:%d/%s?sslmode=%s",
+        url.UserPassword(cfg.db.User, cfg.db.Pass),
+        cfg.db.Driver,
```

Comment:
Should the driver be before the `UserPassword()`?

Reply:
there are different schools of thought on that